README.md

```diff
@@ -1,148 +1,213 @@
-in a distributed design that allows you to query mountains of data in realtime, not batches.
+Crate is a fast, scalable, easy to use SQL database that plays nicely
+with containers like Docker. It feels like the SQL databases you know,
+however makes scaling and operating your database ridiculously easy --
+regardless of the volume, complexity, or type of data. Crate is open
+source. It ingests millions of records per second for time series
+setups and delivers analytics results in subsecond real time.
+
+It comes with a distributed sort and aggregation engine, fast multi
+index queries, native full-text search and super simple scalability
+with sharding and partitioning builtin already. Preconfigured
+replication takes care of data resiliency. The cluster management can
+easily supervised with its builtin admin UI. Crate's masterless
+architecture and simplicity make the data part of Docker environments
+easy and elegant.
+
+Crate provides several installation packages, including a supported
+Docker image. It fits perfectly in an orchestrated microservices
+environment. It acts like an ether, an omnipresent, persistent layer
+for data. This way, application containers access their data
+regardless on which host the data nodes run.
+
+[Crate: Put your data to work. Simply.](https://crate.io/)
+
+## Building Images
+
+Crate derives from
+[alpine:latest](https://hub.docker.com/_/alpine/). We build this
+nightly updated image from the `Dockerfile` on GitHub:
 
-We solve your data scaling problems and make administration a breeze.
-
-**Crate.IO - Put your data to work. Simply.**
-
-[http://crate.io][3]
-
-
-# Crate Dockerfile
-
-This repository contains **Dockerfile** of [Crate][3] for [Docker][1]'s [automated build][2]
-published to the public [Docker Hub Registry][4].
-
-
-## Base Docker Image
-
-- [alpine:latest][5]
-
-## Installation
-
-1. Install [Docker][1]
-
-2. Download latest automated build from public [Docker Hub Registry][2]
+    docker build -t="crate/crate" github.com/crate/docker-crate
 
-    docker pull crate/crate:latest
+To build your own image, replace the image name after the `-t` option
+with your own name.
 
-Alternatively you can build an image from `Dockerfile`:
-
-    docker build -t="crate/crate" github.com/crate/docker-crate
+Alternativly, an [Official Crate
+Image](https://hub.docker.com/_/crate/) with the name `crate` is
+available in the Docker Hub. That image is based on a stable major
+version of Crate, and is tested and scanned by Docker, Inc.
 
 ## How to use this image
 
-    docker run -d -p 4200:4200 -p 4300:4300 crate/crate:latest
+To form a cluster, just start the Crate container a few times in the
+background. This starts a couple of containers on your machine which
+discover each other via multicast. In a production environment you'd
+run Crate on different machines:
 
-### Attach persistent data directory
+```console
+# docker run -d crate/crate
```

Comment:
It is accessible through it's IP. For inter-node communication that work if either a) all nodes are on the same bridge, which is the default on a single host, or b) if you have an overlay network, which is integral part since Docker 1.10. For some (not all) use cases it's ok to have only a single node accessible for clients.